app/models/team.rb

```diff
@@ -150,6 +150,14 @@ def original_team_name
     name.remove(DEACTIVATED_LABEL, /@\((.)*\)/).strip
   end
 
+  def deactivated
+    if active?
+      ""
+    else
+      deleted_at.strftime("%d/%m/%Y")
+    end
+  end
```

Comment:
Nothing wrong with this at all, but just for discussion's sake :) 

Ruby style tends to shy away from explicit indented conditionals and people often try to avoid them - you often see alternative phrasings preferred instead, like a guard clause returning early:

```
  def deactivated
    return "" if active? 
    deleted_at.strftime("%d/%m/%Y")
  end
```
or just sending messages to objects - this example essentially cheats by using the safe navigation operator `&` to hide the if statement and do a concise check for nil, which is what `active?` does  - you'll see this pattern used elsewhere in the file

```
  def deactivated
    deleted_at&.strftime("%d/%m/%Y").to_s
  end
```

Regarding the method name, I'm wondering whether `deactivated` sounds a bit like a boolean - would it make more sense to call it `deactivated_date` or something like that?

Reply:
All very subjective! Final thought, the way it's already done elsewhere in the file is probably clearer than mine, actually...

```  
  def team_leader_name
    team_leader&.value || ''
  end

  def deactivated_date
    deleted_at&.strftime("%d/%m/%Y") || ''
  end
```